### ISSUE TITLE
[BUG] :readonly

### ISSUE THREAD

**Issue by danny007in** [vinorodrigues/bootstrap-dark-5#19]
```bash
 31:4   ✖  Unexpected unknown pseudo-class selector ":readonly"   selector-pseudo-class-no-unknown
```

at forms/_form-control.scss

stylelint error

**Comment by vinorodrigues** [comment]
Ummm.... yes... thanks -- typo on my side :(
Fixed with commit 444883f

**Comment by danny007in** [comment]
I was implemented in https://github.com/ColorlibHQ/AdminLTE/tree/v4-dev , if possible check. 😃

**Comment by danny007in** [comment]
:readonly is there in form_control.scss fix it.
in bootstrap 5.0.1 :readonly reverted
so this dark mode should be replaced with [readonly]

**Comment by vinorodrigues** [comment]
Already fixed in commit [7c4dbb7](https://github.com/vinorodrigues/bootstrap-dark-5/commit/7c4dbb7bc36576de7dd0b5048430ccd469f9b664).
